python/GafferUI/ColorSwatchPlugValueWidget.py

```diff
@@ -110,15 +100,19 @@ def __buttonRelease( self, widget, event ) :
 
 		return True
 
-def _colorFromPlugs( plugs ) :
+def _averageColor( colors ) :
 
-	if not len( plugs ) :
+	if not len( colors ) :
 		return imath.Color4f( 0 )
 
+	return sum( colors ) / len( colors )
+
+def _colorFromPlugs( plugs ) :
+
 	# ColorSwatch only supports one colour, and doesn't have
 	# an "indeterminate" state, so when we have multiple plugs
 	# the best we can do is take an average.
-	return sum( p.getValue() for p in plugs ) / len( plugs )
+	return _averageColor( [ p.getValue() for p in plugs ] )
```

Comment:
This isn't a change in this PR, but I'm unconvinced that returning an average is desirable here.  Something like just defining "invalidColor = Color3f( 1, 0.5, 0 )" and returning that when they don't match might be better?

Reply:
I definitely disagree here. If I select three similarly-coloured lights in the LightEditor, and edit them, it's because I want to give them all the same colour, and there's a high chance that that colour is close to the originals. Starting from arbitrary error colour definitely isn't a help. I could see some sort of `---` overlay being useful, to alert folks to the averaging, but I'd call that out of scope for this PR.